plugins/candela/plugin_tests/candelaSpec.js

```diff
@@ -19,6 +19,10 @@ describe('Test the candela UI.', function () {
         });
         girderTest.waitForLoad();
```

Comment:
Ok this change is in. The first build of this PR failed on installing requirements-dev, hopefully this works.